ios/MmkvHostObject.mm

```diff
@@ -28,11 +35,13 @@
       throw std::runtime_error("Failed to create MMKV instance! `encryptionKey` cannot be longer than 16 bytes!");
     }
 
-    // Check if path is invalid
-    NSFileManager* fileManager = [[NSFileManager alloc] init];
-    bool pathExists = [fileManager fileExistsAtPath:path isDirectory:nil];
-    if (!pathExists) {
-      throw std::runtime_error("Failed to create MMKV instance! The given Storage Path does not exist!");
+    if (appGroup == nil) {
+      // Check if path is invalid
+      NSFileManager* fileManager = [[NSFileManager alloc] init];
+      bool pathExists = [fileManager fileExistsAtPath:path isDirectory:nil];
+      if (!pathExists) {
+        throw std::runtime_error("Failed to create MMKV instance! The given Storage Path does not exist!");
+      }
```

Comment:
Maybe we could also warn to console if the user supplied a custom path and we're using appgroups? Because then path wont work